lib/web_ui/lib/src/engine/profiler.dart

```diff
@@ -106,11 +106,8 @@ class Profiler {
     _checkBenchmarkMode();
 
     final OnBenchmark? onBenchmark =
-        // ignore: implicit_dynamic_function
-        js_util.getProperty(html.window, '_flutter_internal_on_benchmark') as OnBenchmark?;
-    if (onBenchmark != null) {
-      onBenchmark(name, value);
-    }
+        getJsProperty<OnBenchmark?>(html.window, '_flutter_internal_on_benchmark');
+    onBenchmark?.call(name, value);
```

Comment:
It looks like tests [aren't happy](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8829097155180462337/+/u/felt_test:_chrome-unit-linux/stdout) about this change.

Reply:
Done.